<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>555</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoPower V1.2.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{background-color: rgb(255, 255, 255);}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_top" stretch="2,1,4">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_left">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_left" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_taskTypes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>18</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Task</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_left_top">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>12</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_task">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButton_shutdown">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>Shutdown</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_reboot">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>Reboot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_hibernate">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>Hibernate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_sleep">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>Sleep</string>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_closeMonitor">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>Close Monitor</string>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_left_bottom">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_vec">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
border: 0px;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_right">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_right" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_timeSerial">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>18</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_right_top">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>12</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_setTime">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
              </font>
             </property>
             <property name="text">
              <string>Set time</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_dateTime" stretch="3,2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>100</number>
             </property>
             <item>
              <widget class="CustomDateEdit" name="dateEdit">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="maximumDateTime">
                <datetime>
                 <hour>7</hour>
                 <minute>59</minute>
                 <second>59</second>
                 <year>3000</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
               <property name="minimumDateTime">
                <datetime>
                 <hour>8</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>1900</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
               <property name="currentSection">
                <enum>QDateTimeEdit::YearSection</enum>
               </property>
               <property name="displayFormat">
                <string>yyyy/MM/dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="timeSpec">
                <enum>Qt::LocalTime</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="timeEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="displayFormat">
                <string>HH:mm</string>
               </property>
               <property name="timeSpec">
                <enum>Qt::LocalTime</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_right_mindle">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_fromNow">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
              </font>
             </property>
             <property name="text">
              <string>From now on</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_fromNowOn">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>150</number>
             </property>
             <item>
              <widget class="QSpinBox" name="spinBox_hour">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="maximum">
                <number>23</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_hour">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>h</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_space">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>8</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_min">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="maximum">
                <number>59</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_min">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                </font>
               </property>
               <property name="text">
                <string>min</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_right_bottom">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_hor">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
border: 0px;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_bottom">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_bottom" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#widget_bottom{background-color: rgb(242, 243, 243);}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_tips" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_bottom_tip1">
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
               </font>
              </property>
              <property name="text">
               <string>Your device will automatically</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bottom_task">
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(85, 170, 255);</string>
              </property>
              <property name="text">
               <string>Shutdown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bottom_tip_at">
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
               </font>
              </property>
              <property name="text">
               <string>at</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bottom_time">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(85, 170, 255);</string>
              </property>
              <property name="text">
               <string>19:00</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_execute">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
            </font>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">schedule/customdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
